Add count of unread items to RSS tab label

pmzqla opened the pull request.

sledgehammer999 commented on src/gui/mainwindow.cpp:
```diff
@@ -518,6 +519,11 @@ void MainWindow::displayRSSTab(bool enable)
 
+void MainWindow::updateRSSTabLabel(int count)
+{
+    tabs->setTabText(tabs->indexOf(rssWidget), "RSS (" + QString::number(count) + ")");
```
As you said RTL languages. Better use `tr("RSS (%1)").arg(count);`

pmzqla replied:
Right, fixed.

sledgehammer999 commented on src/gui/mainwindow.cpp:
```diff
@@ -518,6 +519,11 @@ void MainWindow::displayRSSTab(bool enable)
 
+void MainWindow::updateRSSTabLabel(int count)
+{
+    tabs->setTabText(tabs->indexOf(rssWidget), QString("RSS (%1)").arg(QString::number(count)));
```
QString::number() isn't needed here. You can pass int directly to arg().

pmzqla replied:
Removed.

pmzqla commented:
Code updated.

pmzqla commented:
Updated. Reason: #3485

sledgehammer999 commented:
This works. However, I don't understand how it fixes your other problem.
Anyway, thanks for the code.

pmzqla commented:
It doesn't. I opened #3485, updated the code here, closed #3485 when I realized it wasn't a good solution and reverted the changes here. I just forgot to update my comment.
